wrappers/python/nntile/nntile_core.cc

```diff
@@ -516,6 +516,15 @@ void def_mod_tensor(py::module_ &m)
     m.def("relu_fp32", &relu<fp32_t>);
     m.def("relu_fp32_fast_tf32", &relu<fp32_fast_tf32_t>);
 
+    // Add activation functions for Tensor<T>
+    m.def("sgd_momentum_async_fp64", &sgd_momentum_async<fp64_t>);
```

Comment:
This function call shall be propagated also into wrappers/python/nntile/functions.py (find adam_step)